### ISSUE TITLE
kubectl install broken on all versions prior to 2.3.2 because of repo name change

### ISSUE THREAD

**Issue by jasgeo75** [oracle-terraform-modules/terraform-oci-oke#206]
https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/fe54d9e4de2d73e81aaa399df71f6b9d8c03d4d9/modules/oke/scripts/install_kubectl.template.sh#L7

Renaming the repo from "ol7_olcne" to  "ol7_olcne11" appears to break kubectl installation on every version prior to 2.3.2.  Was this a necessary change?

**Comment by hyder** [comment]
Hi, 

In a previous release, we had removed the OS version of Oracle Linux altogether to fix an issue when a new minor version is released and the previous minor version is retired. However, last week, OL 8 was made the default version and the oracle-olcne-release-el7 package is only available for Oracle Linux 7. Similarly, the yum channel ol7_olcne11 is only available for Oracle Linux 7. We have added the OS version back in 2.3.2. 

We also need the ol7_olcne11 channel to get the latest version of kubectl. The reason for this is that kubectl is supported within 1 minor version of kube-apiserver.

Our sincere apologies for the inconvenience. We'll look at how we can improve our release mechanism.
